pom.xml

```diff
@@ -75,14 +76,20 @@
             <artifactId>slf4j-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.10.0</version>
```

Comment:
This version can / should be managed in third-party-bom

Reply:
done